app/views/pages/_sidebar.html.erb

```diff
@@ -1,13 +1,23 @@
 <div class="container">
   <h2 style="text-align: center">All Pages</h2>
   <ul class="nav nav-pills nav-fill flex-column">
-    <%- pages.each do |page| %>
-        <%- is_current = page == current_page %>
+    <%- categories.each do |key, value| %>
```

Comment:
I think the arguments should probably be `category, pages` or something like that.